includes/reader-revenue/templates/myaccount-verify.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * My Account page before account has been verified.
+ * The user will be asked to verify before they can manage account settings.
+ *
+ * @package Newspack
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+do_action( 'woocommerce_before_edit_account_form' );
+
+global $newspack_reset_password_arg;
+global $newspack_send_magic_link_arg;
+
+$message = false;
+if ( isset( $_GET['message'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	$message = $_GET['message']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+}
+
+$is_error = false;
+if ( isset( $_GET['is_error'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	$is_error = $_GET['is_error']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
```suggestion
	$is_error = sanitize_text_field( $_GET['is_error'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Reply:
Whoops, `$is_error` is a boolean value, so 44e3abf updates this to use `boolval` instead.